### PR TITLE
[studio-ui] Pasting a large tree takes a while without an indication what is happening #1902

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/1902

### CODE REVIEW

File: static-assets/scripts/main.js [hunk 1]
```diff
@@ -758,7 +758,7 @@
             }
 
             function authLoop() {
-                $scope.authDelay = 1000;
+                $scope.authDelay = 60000;
```
Comment (sumerjabri) [inline static-assets/scripts/main.js:761]: What's this about?

File: ui/scss/temp.scss [hunk 2]
```diff
@@ -92,7 +92,7 @@ body, var {
   top: 0;
   left: 0;
   width: 100%;
-  z-index: 100;
+  z-index: 99;
```
Comment (sumerjabri) [inline ui/scss/temp.scss:95]: Please make sure the session times out, and we gray out the screen, everything is grayed out except the session timeout dialog.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/scss/temp.scss (hunk 2 above)
Comment (jvega190) [reply ui/scss/temp.scss:95]: Yes, actually this change fixes it. Also should have been a separate commit.

File: static-assets/scripts/main.js (hunk 1 above)
Comment (jvega190) [reply static-assets/scripts/main.js:761]: This is a timer for token validation, set to the same time as yui code, should have been a separate commit

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**